app/javascript/components/common/Credits.tsx

```diff
@@ -23,12 +23,12 @@ export function Credits({
 }: CreditsProps): JSX.Element | null {
   const [overflow] = useState<number>(topCount + Number(bottomCount) - 2)
 
-  if (users.length === 0 || !users) {
+  if (!users || users.length === 0) {
     return null
   }
 
   return (
-    <div className={`flex items-center ${className}`}>
+    <div className={`flex items-center ${className ? className : ''}`}>
```

Comment:
Why is this change needed?

Reply:
in this case it fails 'quickerly', and doesn't error out with a `cannot read propertly 'length' of undefined`.

Reply:
I think that was fixed in https://github.com/exercism/website/pull/3159/files#diff-46fb86f1af44dd4f4a5165f8fba3c09d544a6651a560bffb1a9b8c0f5b63e3ecR26. Could you try reverting and see if it works?

Reply:
```suggestion
    <div className={`flex items-center ${className}`}>
```